PowerShell/Add-GraphPermissions-Managed-Identity/Add-GraphPermissions-Managed-Identity.md

```diff
@@ -0,0 +1,42 @@
+# Add-GraphPermissions-Managed-Identity
```

Comment:
Can you rename the folder/files to Add-GraphPermissionsToManagedIdentity

Reply:
Resolved